src/restapi/fConfig.ts

```diff
@@ -35,6 +36,16 @@ export async function fSubmitOrUpdateZendeskConfigSubmit(req: Request, res: Resp
         const targetID = cValues.zd_target_id;
         const zdOauth2AccessToken = cValues.zd_oauth_access_token;
         const storeValues = call.values as AppConfigStore;
+        storeValues.zd_url = storeValues.zd_url.replace(/\/$|(?<!\/)\/+$/, '');
```

Comment:
Maybe we should have a comment saying exactly what the goal of the regex is. I have to say that it's gotten a little more complicated and I wouldn't know what it's doing upon inspection

Reply:
@mickmister I added a pretty length comment with a link to the example. This was actually a pretty useful catch by codeQL